java/runtime-common/src/main/java/org/ray/spi/LocalSchedulerLink.java

```diff
@@ -15,19 +15,19 @@
 
   void markTaskPutDependency(UniqueID taskId, UniqueID objectId);
 
-  void reconstructObject(UniqueID objectId);
+  void reconstructObject(UniqueID objectId, boolean fetchOnly);
 
   void notifyUnblocked();
 
   List<byte[]> wait(byte[][] objectIds, int timeoutMs, int numReturns);
 
-  default void fetch(UniqueID objectId) {
-    reconstructObject(objectId);
+  default void fetch(UniqueID objectId, boolean fetchOnly) {
```

Comment:
This method shouldn't have the `fetchOnly` parameter.

In Raylet's local scheduler client, `fetch = reconstruct_object(fetch_only=True)`. When `fetch_only=False`, `reconstruct` = `reconstruct and fetch`.

I found the meaning of the this function is confusing, maybe we should separate them to 2 functions `fetch` and `reconstruct_and_fetch`? @robertnishihara